packages/components/code-quality/html-validate/scripts/synergy-element-rules.js

```diff
@@ -0,0 +1,437 @@
+ */
+const SynDivider = {
+  flow: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynDrawer = {
+  focusable: true,
+  sectioning: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynDropdown = {
+  flow: true,
+  focusable: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynFile = {
+  flow: true,
+  focusable: true,
+  formAssociated,
+  interactive: true,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynHeader = {
+  attributes: {
+    'show-burger-menu': {
+      allowed: deprecatedForV2('Please use the `burger-menu` attribute instead.'),
+      boolean: true,
+    },
+  },
+  flow: true,
+  heading: true,
+  sectioning: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynIconButton = {
+  focusable: true,
+  formAssociated,
+  interactive: true,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynIcon = {
+  embedded: true,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynInput = {
+  flow: true,
+  focusable: true,
+  formAssociated,
+  interactive: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynMenuItem = {
+  focusable: true,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynMenuLabel = {
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynMenu = {
+  focusable: true,
+  sectioning: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynNavItem = {
+  focusable: true,
+  interactive: true,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynOptgroup = {};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynOption = {
+  focusable: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynPrioNav = {
+  focusable: true,
+  sectioning: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynProgressBar = {
+  formAssociated,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynProgressRing = {
+  formAssociated,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynRadioGroup = {
+  flow: true,
+  focusable: true,
+  formAssociated,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynRadio = {
+  focusable: true,
+  formAssociated,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynRangeTick = {
+  flow: true,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynRange = {
+  flow: true,
+  focusable: true,
+  formAssociated,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynSelect = {
+  flow: true,
+  focusable: true,
+  formAssociated,
```

Comment:
I guess should be also interactive

```suggestion
  formAssociated,
  interactive: true,
```

Reply:
Fixed in latest